crates/hyperdrive-test-utils/src/chain/deploy.rs

```diff
@@ -919,7 +922,6 @@ impl TestnetDeploy for Chain {
         base.transfer_ownership(config.admin).send().await?;
         vault.transfer_ownership(config.admin).send().await?;
         lido.transfer_ownership(config.admin).send().await?;
-        factory.update_governance(config.admin).send().await?;
```

Comment:
I could be missing something, but I don't think we want to remove this. What was the impetus for this change?

Reply:
The impetus is that `UpdateGovernance` is gone now. and my understanding is we only have to do UpdateAdmin in the registry and not the factory.

Reply:
I'm not sure what you mean. It's still there on main: https://github.com/delvtech/hyperdrive/blob/main/contracts/src/factory/HyperdriveFactory.sol#L354.

Reply:
got it, I was mixing up where that was deleted. I added it back in.